src/model.rs

```diff
@@ -348,3 +348,8 @@ impl ops::Deref for Model {
         &self.collections
     }
 }
+impl ops::DerefMut for Model {
+    fn deref_mut(&mut self) -> &mut Collections {
+        &mut self.collections
+    }
+}
```

Comment:
No, you can't do that. The point of Model only implementing Deref is because modifying the Collections will invalidate the model.

Reply:
You still need to remove this impl.

Reply:
oups sorry :D